infra/registry/output.tf

```diff
@@ -0,0 +1,24 @@
+output "vm-name" {
+  value = "${openstack_compute_instance_v2.instance[*].name}"
+}
+ 
+output "vm-id" {
+  value = "${openstack_compute_instance_v2.instance[*].id}"
+}
+ 
+output "vm-ips" {
+  value = "${openstack_compute_instance_v2.instance[*].network[0].fixed_ip_v4}"
+}
+
+output "subnt_1" {
+  value = "${openstack_networking_subnet_v2.subnet[*].id}"
+}
+
+output "vm-floating-ip" {
+  value = openstack_networking_floatingip_v2.fip[*].address
+}
+
+output "lb_loadbalancer_id" {
+  value = openstack_lb_loadbalancer_v2.lb.id
+}
+
```

Comment:
Why do we need al these outputs?

Reply:
There isno need in these outputs, except vm-floating-ip. This output will be used to deploy docker-registries with ansible

Reply:
ok, remove all unused outputs
either add a note to readme on how to invoke ansible-playbook with terraform outputs
or don't use terraform outputs at all and use static ansible inventory with DNS instead of IPs